test/board_test.exs

```diff
@@ -26,4 +26,24 @@ defmodule BoardTest do
 
     assert board == expected_board
   end
+
+  test "determines noone has won" do
+    board = %{
+      1 => "X", 2 => "2", 3 => "X",
+      4 => "O", 5 => "5", 6 => "O",
+      7 => "X", 8 => "8", 9 => "O"
+    }
+
+    assert Board.has_player_won(board) === false
+  end
+
+  test "determines X is the winner" do
+    board = %{
+      1 => "X", 2 => "2", 3 => "X",
+      4 => "O", 5 => "X", 6 => "O",
+      7 => "X", 8 => "8", 9 => "O"
+    }
+
+    assert Board.has_player_won(board) === true
```

Comment:
I would recommend adding describe blocks for each of your public functions since your `Board` module has a handful of them.

Reply:
I think this makes sense to me, so a describe block for each test in `board_test` essentially? Just want to clarify before!

Reply:
Not each test, just each public function. `"describe has_player_won?/1"`, `"describe place_a_symbol/3"` – with the tests corresponding with each function nested inside of that describe block.